src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -87,8 +87,13 @@ const DragDropcontext: FC<dragDropProps> = ({
         const assignee = result.combine.droppableId === 'tasks'
           ? result.draggableId
           : result.combine.draggableId;
+        const dateObject:any = new Date();
+        const startedOn:any = dateObject / 1000;
```

Comment:
- I think it'll be a good idea to have a variable for `1000`, it'll be helpful to understand why are we dividing the `dateObject` by `1000`.

- I think we need to get epoch from the date before performing the division. e.g. `dateObject.getTime() / 1000`

I am getting this error in `tsserver - eslint`
> The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.

Reply:
Got it
> I will have a separate variable for 1000
> `I think we need to get epoch from the date before performing the division. e.g. dateObject.getTime() / 1000` :- I also thought so but then tried it and it was working, but anyway its good to avoid confusions will update it